bubblewrap.c

```diff
@@ -439,6 +439,19 @@ has_caps (void)
   return data[0].permitted != 0 || data[1].permitted != 0;
 }
 
+static void
+drop_cap_bounding_set (void)
+{
+  unsigned long cap;
+
+  for (cap = 0; cap <= 63; cap++)
```

Comment:
The syscall api used to set capabilities currently allow a maximum of 64 capabilities (its got 2 32bit fields). There are not actually that many, but we just ignore EINVAL for the "too large" ones.

Eventually the kernel could have some new syscall to get more caps, but I doubt that will happen.